### PR TITLE
Adds tests for ExpandedListItems

### PR DESCRIPTION
**Pull request by limbovski**

### CODE REVIEW

File: src/components/Sidebar/components/ExpandedList/ExpandedList.test.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+/*
+// jest.it and jest.test are aliases of the same function
+// But typically you would use it as per the example below
+// whereas test would be used if you word the function as follows:
+// test("<ExpandedList", () => {
+// // Test goes here  
+// });
+// Useful link https://blog.sapegin.me/all/react-testing-3-jest-and-react-testing-library/
+*/
```
Comment (MoSattler) [inline src/components/Sidebar/components/ExpandedList/ExpandedList.test.tsx:13]: lookgs good, but htat comment would be better inside hte PR as a comment, not in the code

File: src/components/Sidebar/components/UserCard/ExpandedList.test.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  );
+
+  expect(queryByText("Profile & Settings")).not.toBeInTheDocument(); // not expanded yet, should be invisible
+
+  userEvent.click(queryByText(testUser.firstName)); // now I clicked on the card (in this case firstName, but could have been anything else)
+  expect(queryByText("Profile & Settings")).toBeInTheDocument(); // now it should be visible
+
+  userEvent.click(queryByText(testUser.firstName)); // now I clicked on the card (in this case firstName, but could have been anything else)
+  expect(queryByText("Profile & Settings")).not.toBeInTheDocument(); // collapsed again, should be invisible
+});
```
Comment (MoSattler) [inline src/components/Sidebar/components/UserCard/ExpandedList.test.js:69]: Did you make a copy of UserCard.test.tsx for some reason?

File: src/components/Sidebar/components/ExpandedList/ExpandedList.test.tsx (hunk 1 above)
Comment (limbovski) [reply src/components/Sidebar/components/ExpandedList/ExpandedList.test.tsx]: Sorted.

File: src/components/Sidebar/components/UserCard/ExpandedList.test.js (hunk 2 above)
Comment (limbovski) [reply src/components/Sidebar/components/UserCard/ExpandedList.test.js]: Ah yes while learning. That file has been sent to the shadow realm now.

**Comment by MoSattler** [comment]
@all-contributors please add @ruaridhm  for test

**Pull request merged**